### ISSUE TITLE
Messages not being extracted when using React hook only

### ISSUE THREAD

**Issue by ramsy-leftclick** [formatjs/formatjs#605]
**Which package?**
babel-plugin-react-intl

**Describe the bug**
When I use the React hook for translating my text in components directly the messages do not get extracted.

**To Reproduce**
1. Use the following code in your component
```
const {formatMessage: f} = useIntl()
...
return <span>{f({id: 'notExtracted', defaultMessage: 'This is not extracted'})}</span>
...
```
2. Run babel-plugin-react-intl

**Expected behavior**
I expected these messages to be extracted in the configured folder (eg. /build/messages/src/...)

**Screenshots**
n/a

**Desktop (please complete the following information):**

- OS: Latest Ubuntu
- Browser: Latest Firefox

**Additional context**
When using the <FormatMessage /> component the messages get extracted successfully.

**Comment by longlho** [comment]
yes we right now do not support scraping `formatMessage` alias to another function right now.

**Comment by mcalthrop** [comment]
This also applies to the [`formatjs` CLI](https://github.com/formatjs/formatjs/blob/master/website/docs/tooling/cli.md) when using the `useIntl` React hook of course.

Example (contrived) of code for which the `formatjs` cli does not extract translation id:

```ts
import React from 'react';
import { useIntl } from 'react-intl';

const MyComponent = () => {
  const { formatMessage } = useIntl();
  const altText = formatMessage({
    id: 'page.image.altText',
    defaultMessage: 'image text for accessibility',
  });
  return <img src="/path/to/image.png" alt={altText} />;
};
```
